### PR TITLE
[WIP] SDK node rename to Framework

### PR DESCRIPTION
**Pull request by etbyrd**
**Customer scenario**

N/A

**Bugs this fixes:** 

Part of https://github.com/dotnet/project-system/issues/2248

**Workarounds, if any**

N/A

**Risk**

Low, just renaming of files and the SDK node

**Performance impact**

None, just renaming

**Is this a regression from a previous update?**

No

**Root cause analysis:**

N/A

**How was the bug found?**

Requests from team members

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx [hunk 1]
```diff
@@ -179,7 +179,7 @@ Press Ignore to ignore the external changes. Your changes may be lost if you clo
     <value>COM</value>
   </data>
   <data name="SdkNodeName" xml:space="preserve">
```
Comment (Pilchie) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx]: Should probably rename this as well.

**Comment by etbyrd** [comment]
retest windows_debug please

**Comment by Nirmal4G** [comment]
Why rename the Sdk node, the previous issue comments show that we need both SDK node and the Framework Node, so Why not keep the SDK node and duplicate the SDK node into Framework node and put the SDK node under a feature flag or something so that we can enable them when the SDK resolvers implement them?!

It's just a waste to throw all of the work away!

**Comment by jmarolf** [comment]
@Nirmal4G this is a point-in-time thing.  We have no intention of releasing `Framework` without `SDK`.  It's just that `Framework` is a rather obvious name change compared to `SDK` which has more design considerations. No need to clutter this PR with `SDK` node design discussions.

**Comment by davkean** [comment]
We're going to approach this a different way; we are adding a framework node.

**Pull request closed without merging**